### ISSUE TITLE
[Mobile] The user can't navigate through the menu

### ISSUE THREAD

**Issue by ovidiuboca1** [mozilla/sumo-project#284]
Prerequisites: Use a mobile device. (I used iPhone 8 plus)

STR:

1. Go to https://support.allizom.org/en-US/
2. Open the menu and try to scroll to the bottom to see all the available options. 

ER: The user can navigate through the menu.
AR: The menu it's closing when the user tries to navigate through the menu. See the screen recording - https://streamable.com/as2f9
Note: Comparing the current behavior from staging with the one from test env. I think this is a regression.

**Comment by ovidiuboca1** [comment]
duplicate in bugzilla https://bugzilla.mozilla.org/show_bug.cgi?id=1623781
I filed this bug to keep the track also in Github.

**Comment by LeoMcA** [comment]
Duplicate of #273
